crates/energy/Cargo.toml

```diff
@@ -14,4 +14,13 @@ categories.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+de_spawner.workspace = true
```

Comment:
Please separate DE & other dependendics and make the groups sorted. Camera example:

```toml
# DE
de_conf.workspace = true
de_core.workspace = true
de_map.workspace = true
de_terrain.workspace = true
de_uom.workspace = true

# Other
bevy.workspace = true
parry3d.workspace = true
```